msticnb/nb/azsent/alert/ti_enrich.py

```diff
@@ -0,0 +1,333 @@
+# -------------------------------------------------------------------------
```

Comment:
I guess it goes without saying that you should implement the new patterns. Shouldn't affect too much of the code